packages/react-components/src/components/MessageThread.tsx

```diff
@@ -869,14 +871,16 @@ export const MessageThread = (props: MessageThreadProps): JSX.Element => {
           <ChatMessageComponent
             {...messageProps}
             message={messageProps.message}
+            // -1 because participant count does not include myself
```

Comment:
I'd drop this comment. The prop is named very well, so it's obvious why you need to subtract 1.